<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackWidget</class>
 <widget class="QWidget" name="PackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="pack_name">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pack_more">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="browser"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pack_install">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="down">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
